mixitup/index.html

```diff
@@ -663,142 +77,29 @@ <h3 class="nameOnModal">Priyanka Oberoi</h3>
                     <div class="row">
                         <div class="col-lg-4 col-md-4 col-sm-4">
                             
-                            <img class="img-responsive img-centered img-circle" src="BioPics/Burton.jpg" alt="Burton Reist">
+                            <img class="img-responsive img-centered img-circle" src="BioPics/{{picture_file}}" alt="{{first_name}} {{last_name}}">
                             
                         </div>
                         <div class="col-lg-7  col-md-7 modalTextStyles" style="text-align:left; color: black">
 
                                 <!-- Bio Details Go Here -->
 
-                                <h3 class="nameOnModal">Burton Reist</h3>
-                                <h4>Director of External Affairs</h4>
+                                <h3 class="nameOnModal">{{first_name}} {{last_name}}</h3>
```

Comment:
I get what you're saying but is there a reason to do that rather than using a class name? I'm only asking because @radhikabhatt26 and I were talking about this last week and wondering if there was a best practice. I figured classes were better b/c they can target specific content - but if using child selectors is better I'm totally down for doing that too!